src/content/docs/apm/agents/net-agent/troubleshooting/lock-contention-appdomain-getdata.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: High AppDomain.GetData() lock contention (.NET)
+type: troubleshooting
+tags:
+  - Agents
+  - NET agent
+  - Troubleshooting
+translate:
+  - jp
+metaDescription: Instructions and information about reducing AppDomain.GetData() lock contention for impacted .NET Framework applications 
+redirects:
+  - /docs/agents/net-agent/troubleshooting/lock-contention-appdomain-getdata
+---
+
+## Problem
+
+When the .NET agent is attached to high throughput **.NET Framework** applications, some customers have reported that thread lock contention can increase dramatically to a point that the application becomes non-responsive. Upon deeper analysis with process dumps, many threads can be blocked in calls to AppDomain.GetData() that are introduced by the .NET Agent.
```

Comment:
Please reword:
some customers have reported that thread lock contention can increase dramatically to a point that the application becomes non-responsive.
**to**
a few customers reported observing high thread contention.  Upon  reviewing the process dumps and stack traces, many threads may be blocked by calls to AppDomain.GetData( ).